src/Generator/ClassGenerator.php

```diff
@@ -725,13 +727,7 @@ public function getProperty($propertyName)
         return false;
     }
 
-    /**
-     * Add a class to "use" classes
-     *
-     * @param  string $use
-     * @param  string|null $useAlias
-     * @return static
-     */
+    /** {@inheritDoc} */
```

Comment:
```suggestion
    /** @inheritDoc */
```

Since this should be importing the whole docBlock, I would make it explicit https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc-tags.md#41-making-inheritance-explicit-using-the-inheritdoc-tag.